Formula/jmeter.rb

```diff
@@ -11,34 +11,9 @@ class Jmeter < Formula
     sha256 "be9973637adf70706a9e596b6d8098e3cb6d99757177d07c81a502da8ae6c719" => :high_sierra
   end
 
-  resource "jmeterplugins-standard" do
-    url "https://jmeter-plugins.org/downloads/file/JMeterPlugins-Standard-1.4.0.zip"
-    sha256 "3f740bb9b9a7120ed72548071cd46a5f92929e1ab196acc1b2548549090a2148"
-  end
-
-  resource "serveragent" do
-    url "https://jmeter-plugins.org/downloads/file/ServerAgent-2.2.1.zip"
-    sha256 "2d5cfd6d579acfb89bf16b0cbce01c8817cba52ab99b3fca937776a72a8f95ec"
-  end
-
-  resource "jmeterplugins-extras" do
-    url "https://jmeter-plugins.org/downloads/file/JMeterPlugins-Extras-1.4.0.zip"
-    sha256 "de35e653250882268aa24d011ec0f2afbc13e1c552fbb676c67515bc80ef3194"
-  end
-
-  resource "jmeterplugins-extraslibs" do
-    url "https://jmeter-plugins.org/downloads/file/JMeterPlugins-ExtrasLibs-1.4.0.zip"
-    sha256 "81d600a5bda6fdb362573d55c11208b2635728a2c18b7f647b9c7413c0f33ef3"
-  end
-
-  resource "jmeterplugins-webdriver" do
-    url "https://jmeter-plugins.org/downloads/file/JMeterPlugins-WebDriver-1.4.0.zip"
-    sha256 "521c2f7d452a84099407534bd50f29fd3761aa8a5beca52966bb9731e33b03e2"
-  end
-
-  resource "jmeterplugins-hadoop" do
-    url "https://jmeter-plugins.org/downloads/file/JMeterPlugins-Hadoop-1.4.0.zip"
-    sha256 "93030738d613748a685764fbfff0fe00ad2e161f2b72df6365294adc88db09b4"
+  resource "jmeter-plugins-manager" do
+    url "https://jmeter-plugins.org/get/"
```

Comment:
Use `https://search.maven.org/remotecontent?filepath=kg/apc/jmeter-plugins-manager/1.3/jmeter-plugins-manager-1.3.jar` plase

Reply:
sure!
how do you find that url?

Reply:
https://jmeter-plugins.org/get/ gives you a file from https://maven.org, so I just searched for `jmeter-plugins-manager` on https://search.maven.org/